devsetup/scripts/standalone.sh

```diff
@@ -112,15 +116,16 @@ fi
 # Use os-net-config to add VLAN interfaces which connect edpm-compute-0 to the isolated networks configured by install_yamls.
 sudo mkdir -p /etc/os-net-config
 
-cat << __EOF__ | sudo tee /etc/cloud/cloud.cfg.d/99-edpm-disable-network-config.cfg
-network:
-    config: disabled
-__EOF__
-
-sudo systemctl enable network
-sudo cp /tmp/net_config.yaml /etc/os-net-config/config.yaml
-sudo os-net-config -c /etc/os-net-config/config.yaml
+if [[ \$STANDALONE_VM == "true" ]]; then
+    cat << __EOF__ | sudo tee /etc/cloud/cloud.cfg.d/99-edpm-disable-network-config.cfg
+    network:
+        config: disabled
+    __EOF__
```

Comment:
need to update this and remove the indendation of __EOF

Reply:
there https://github.com/openstack-k8s-operators/install_yamls/pull/593#event-10870059985